tutorials/xr/xr_action_map.rst

```diff
@@ -385,3 +385,124 @@ This is why many XR runtimes only use it as a last resort and will attempt to us
+    A value above ``90 degrees`` means that the slices overlap and that multiple actions can be in the pressed state.
+  * ``Is Sticky``, when enabled means that an action stays in the pressed state until the thumbstick or trackpad moves into another wedge even if it has left the wedge for that action.
+  * ``On Haptic`` lets us define a haptic output that is automatically activated when an action becomes pressed.
+  * ``Off Haptic`` lets us define a haptic output that is automatically activated when a action is released.
+
+
+Binding modifiers on individual bindings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Binding modifiers that are applied to individual bindings can be accessed through the binding modifier button next to action attached to an input:
+
+.. image:: img/openxr_action_binding_modifier.webp
+
+You can add a new modifier by pressing the ``Add binding modifier`` button.
+
+.. warning::
+  As Godot doesn't know which inputs on each runtime support a modifier,
+  there is no restriction to adding modifiers.
+  If the modifier extension is unsupported, modifiers will be filtered out at runtime.
+  Modifiers added to the wrong input may result in a runtime error.
+
+  You should test your action map on the actual hardware and runtime to verify the proper setup.
+
+Analog threshold modifier
+"""""""""""""""""""""""""
+
+The analog threshold modifier allows you to specify the thresholds used for any analog input, like the trigger, that has a boolean input. This controls when the input is in the pressed state.
+
+In order to use this modifier you must enable the analog threshold extension in the project settings:
+
+.. image:: img/openxr_project_settings_analog_threshold_modifier.webp
+
+The analog threshold modifier has the following settings:
+
+.. image:: img/openxr_analog_threshold_modifier.webp
+
+These are defined as follows:
+
+  * ``On Threshold`` specifies the threshold value that will enable the action, e.g. a value of ``0.6`` means that when the analog value gets above ``0.6`` the action is set to the pressed state.
+  * ``Off Threshold`` specifies the threshold value that will disable the action, e.g. a value of ``0.4`` means that when the analog value goes below ``0.4`` the action is set in to the released state.
+  * ``On Haptic`` lets us define a haptic output that is automatically activated when the input is pressed.
+  * ``Off Haptic`` lets us define a haptic output that is automatically activated when the input is released.
+
+Haptics on modifiers
+^^^^^^^^^^^^^^^^^^^^
+
+Modifiers can support automatic haptic output that is triggered when thresholds are reached.
+
+.. note::
+  Currently both available modifiers support this feature however there is no rule future modifiers also have this capability.
+  Only one type of haptic feedback is supported but in the future other options may become available.
+
+Haptic vibration
+================
```

Comment:
```suggestion
""""""""""""""
```
`===` is already used for the title of this page. I think you mean to use `"""` here, judging by existing usages